Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -420,7 +419,7 @@ Worker_CommandRequest USpatialSender::CreateRPCCommandRequest(UObject* TargetObj
 		return CommandRequest;
 	}
 
-	Schema_AddPayload(RequestObject, 2, PayloadWriter);
+	Schema_AddPayload(RequestObject, 1, PayloadWriter);
```

Comment:
Josh was saying we should make these indices into named constants. Would you want to do that here or in a different PR?

Reply:
Different PR.